Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -359,6 +365,70 @@ public class EOSSessionsManager : IEOSSubManager
 
         private bool subscribtedToGameInvites = false;
         private bool userLoggedIn = false;
+
+        #region Peer2Peer Messaging Variables
+        /// <summary>
+        /// When sending or receiving Peer2Peer messages relating to Session Status upkeep, always use this same socket name.
+        /// This should be a distinct and unique socket name, different than any other socket name utilized in the program.
+        /// </summary>
+        private const string P2PSessionStatusSocketName = "SESSIONSTATUS";
+
+        /// <summary>
+        /// When sending or receiving Peer2Peer messages relating to Session status upkeep, always use this same channel.
+        /// This should be a distinct and unique channel, different than any other socket name utilized in the program, if possible.
```

Comment:
It _appears_ as though you can have a socket name collision, as long as they're on different channels. Because `HandleReceivedP2PMessages` uses only the channel in the `ReceivePacketOptions` and not the socket to get incoming messages, it's most important that the channel is unique. Otherwise someone implementing this could receive messages meant for another socket in this method, or get messages meant for session management in another method.

I'm noticing in this comment that it's a little unclear, so we're gonna update this text a bit.